### PR TITLE
Postgres db

### PR DESCRIPTION
**Pull request by kristoff-campbell28**
Making this PR to allow comparison b/w this branch and `master`, requested.

### CODE REVIEW

File: creation/views.py [hunk 1]
```diff
@@ -12,11 +12,11 @@ def create(request):
 
         if form.is_valid():
             event = form.save(commit=False)
-            print(request)
             event.created_date = timezone.now()
             event.access_code = genAccessCode()
-
             event.save()
+            p = Group(event=event)
```
Comment (KevinCybura) [inline creation/views.py]: We should be creating a Group here we will be creating them later when we start letting user join the event which is what we will be working next.

File: event/views.py [hunk 2]
```diff
@@ -1,9 +1,39 @@
+from django.core.exceptions import ObjectDoesNotExist
 from django.http import HttpResponseRedirect
 from django.shortcuts import render
-from creation.models import Event 
+from creation.models import Event, Group, User
+from Home.forms import Join
+
+def join(request):
+    if request.method == 'POST':
```
Comment (KevinCybura) [inline event/views.py]: The join view is already in the Home app so we don't need it here and dont worry about dividing the groups I believe Silvena is working on that.

File: Home/urls.py [hunk 3]
```diff
@@ -1,9 +1,27 @@
...
+    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
+"""
+
 from django.urls import path, include
 from . import views
 from django.contrib.auth import views as auth_views
 
 urlpatterns = [
     path('', views.home, name="home_creation_page"),
+	path('create/', include('creation.urls')),
```
Comment (Dandanlin0702) [inline Home/urls.py]: is this creations.url for game creation? or group creations?

File: creation/models.py [hunk 4]
```diff
@@ -1,24 +1,28 @@
...
             default=timezone.now) 
     access_code = models.CharField(max_length=8)
+	
+    def __str__(self):
+        return "{}: {} ".format(self.title, self.description)
+
+class Group(models.Model):
+    event = models.ForeignKey(Event, on_delete = models.CASCADE)
+    curr_size = models.IntegerField(default=0)
+    max_size = models.IntegerField(default=4)
```
Comment (KevinCybura) [inline creation/models.py]: Should add for now `users = models.ForeignKey(User, on_delete=models. DO_NOTHING)`.
Also you will have to put `from django.contrib.auth.models import User` at the top of them file.

Do the same thing for the Event model

File: creation/models.py [hunk 5]
```diff
@@ -1,24 +1,28 @@
...
 
 class Event(models.Model):
     title = models.CharField(max_length=255)
-    #admin = models.ForeignKey(User, on_delete=models.SET_NULL, null=True, related_name="event_admin")
+    # admin = models.ForeignKey(User, on_delete=models.SET_NULL, null=True, related_name="event_admin")
     admin = models.CharField(max_length=100);
     description = models.CharField(default="", max_length=500)
     event_type = models.CharField(max_length=100)
-    users = models.ManyToManyField(User, related_name="event_user")
+    # users = models.ManyToManyField(User, related_name="event_user")
```
Comment (KevinCybura) [inline creation/models.py]: See Group model comment

File: Home/templates/Home/game.html [hunk 6]
```diff
@@ -1,5 +1,24 @@
...
+			// Handler for .ready() called.
+		});
+	</script>
+    <header class="masthead bg-primary text-white">
+      <div class="container-fluid">
+        {% if game %}
+          <h2 class="text-center text-uppercase" id="game">{{ game }}</h2>
+          <hr class="star-light">
+=======
+>>>>>>> da4b183bf1aa60df1094ce0673e7c9ff3ba83ef0
```
Comment (Dandanlin0702) [inline Home/templates/Home/game.html]: conflict here

File: Home/urls.py (hunk 3 above)
Comment (KevinCybura) [inline Home/urls.py]: Dont think we need this here the `create` view will be called through `creation/create`

File: Home/templates/Home/game.html [hunk 7]
```diff
@@ -1,5 +1,24 @@
 {% extends "Home/base.html" %}
 {% load staticfiles %}
+<<<<<<< HEAD
```
Comment (KevinCybura) [inline Home/templates/Home/game.html]: This is a merge conflict that need to be fixed

File: creation/models.py [hunk 8]
```diff
@@ -1,24 +1,28 @@
...
+    def __str__(self):
+        return "{}: {} ".format(self.title, self.description)
+
+class Group(models.Model):
+    event = models.ForeignKey(Event, on_delete = models.CASCADE)
+    curr_size = models.IntegerField(default=0)
+    max_size = models.IntegerField(default=4)
+    
+class User(models.Model):
+    group = models.ForeignKey(Group, on_delete = models.CASCADE, default = "")
```
Comment (KevinCybura) [inline creation/models.py]: We should delete this now because we decided to use the default django user.

File: event/urls.py [hunk 9]
```diff
@@ -2,6 +2,8 @@
 from . import views
 # from django.conf.urls import url
 from . import views
+
 urlpatterns = [
+    path('', views.join, name="join_page"),
```
Comment (KevinCybura) [inline event/urls.py]: Join page should stay in the Home app see comment below

File: Home/urls.py (hunk 3 above)
Comment (kristoff-campbell28) [reply Home/urls.py]: Group creations; i moved most of the creation stuff to the actual creations app for order's sake.

File: creation/models.py (hunk 4 above)
Comment (kristoff-campbell28) [reply creation/models.py]: Added the import, but isn't the "many" end of a many-to-one relationship supposed to have the foreign key?

File: creation/models.py (hunk 5 above)
Comment (kristoff-campbell28) [reply creation/models.py]: see response above

File: creation/models.py (hunk 4 above)
Comment (KevinCybura) [reply creation/models.py]: Not sure havent worked with database all that much but we need the Group to have multiple user.

Comment (kristoff-campbell28) [reply creation/models.py]: yeah, that's what I mean. Every user stores the `id` of a Group object to show what group it's part so, and when we call `Group.user_set()` that'll return all the users in a group.

**Pull request closed without merging**